commands/list_releases.go

```diff
@@ -96,58 +93,116 @@ func listReleasesPreconditions(args *listReleasesArguments) error {
 // errors in case they happen
 func listReleasesRunOutput(cmd *cobra.Command, extraArgs []string) {
 	args := defaultListReleasesArguments()
-	result, err := listReleases(args)
+	releases, err := listReleases(args)
 
-	// error output
 	if err != nil {
 		handleCommonErrors(err)
 
-		var headline = err.Error()
-
-		fmt.Println(color.RedString(headline))
+		if clientErr, ok := err.(*clienterror.APIError); ok {
+			fmt.Println(color.RedString(clientErr.ErrorMessage))
+			if clientErr.ErrorDetails != "" {
+				fmt.Println(clientErr.ErrorDetails)
+			}
+		} else {
+			fmt.Println(color.RedString("Error: %s", err.Error()))
+		}
 		os.Exit(1)
 	}
 
-	// success output
-	if len(result.releases) == 0 {
+	// success
+	if len(releases) == 0 {
 		fmt.Println(color.RedString("No releases available."))
 		fmt.Println("We cannot find any releases. Please contact the Giant Swarm support team to find out if there is a problem to be solved.")
-		os.Exit(1)
-	} else {
+	}
 
-		for _, release := range result.releases {
+	// table headers
+	output := []string{strings.Join([]string{
+		color.CyanString("VERSION"),
+		color.CyanString("STATUS"),
+		color.CyanString("KUBERNETES"),
+		color.CyanString("CONTAINERLINUX"),
+		color.CyanString("COREDNS"),
+		color.CyanString("CALICO"),
+		color.CyanString("CREATED"),
```

Comment:
I would move `CREATED` to the third position

Reply:
I had this in the first place. But that isn't easy to parse and imo it is "learned" that such dates are at the end of the line. But thats your call. You still prefer it?

![screen_2018-10-04-09-37-46](https://user-images.githubusercontent.com/32488/46459577-98215a80-c7b9-11e8-8a5e-a7e1672b6ffd.png)

Reply:
Maybe if I remove the time? Not sure if the time is really valuable information and we still display the exact time in `show release`